README.md

```diff
@@ -1,17 +1,5 @@
 # tripleawarclub.org
 
-PHP source code repository 
+This repository contains the nginx configuration file for \[www.\]tripleawarclub.org.
 
-
-## Community Updates and Deployment
-
-PRs are welcome. 
-
-
-## Dev Setup
-
-We could use some help documenting setup instructions. Should be a standard apache deployment of the 'tripleawarclub.org' folder to `/var/www/` . DB will need to be setup. DB creds are in two files: secure.php and mainfile.php
-
-## Website Future
-
-There have been a number of conversations about the warclub website. Veqryn's advice is to consider upgrading to a more modern tech stack. Meanwhile, there is interest in keeping the current website features working.
+This site is no longer active and therefore it permanently redirects to [forums.triplea-game.org](forums.triplea-game.org).
```

Comment:
I think you need to include the `https://` scheme on the URL.  Otherwise, the markdown renderer treats it as relative to whatever URL you're currently viewing (e.g. the commit or PR URL).